RecoLocalTracker/SiPixelRecHits/src/PixelCPEFastParamsHost.cc

```diff
@@ -347,37 +369,44 @@ void PixelCPEFastParamsHost<TrackerTraits>::errorFromTemplates(DetParam const& t
   theClusterParam.pixmx = std::numeric_limits<int>::max();  // max pixel charge for truncation of 2-D cluster
 
   theClusterParam.sigmay = -999.9;  // CPE Generic y-error for multi-pixel cluster
+  theClusterParam.sigmay = -999.9;  // CPE Generic y-error for multi-pixel cluster
+  theClusterParam.deltay = -999.9;  // CPE Generic y-bias for multi-pixel cluster
   theClusterParam.sigmax = -999.9;  // CPE Generic x-error for multi-pixel cluster
+  theClusterParam.deltax = -999.9;  // CPE Generic x-bias for multi-pixel cluster
   theClusterParam.sy1 = -999.9;     // CPE Generic y-error for single single-pixel
+  theClusterParam.dy1 = -999.9;     // CPE Generic y-bias for single single-pixel cluster
   theClusterParam.sy2 = -999.9;     // CPE Generic y-error for single double-pixel cluster
+  theClusterParam.dy2 = -999.9;     // CPE Generic y-bias for single double-pixel cluster
   theClusterParam.sx1 = -999.9;     // CPE Generic x-error for single single-pixel cluster
+  theClusterParam.dx1 = -999.9;     // CPE Generic x-bias for single single-pixel cluster
   theClusterParam.sx2 = -999.9;     // CPE Generic x-error for single double-pixel cluster
-
-  float dummy;
+  theClusterParam.dx2 = -999.9;     // CPE Generic x-bias for single double-pixel cluster
 
   SiPixelGenError gtempl(this->thePixelGenError_);
   int gtemplID = theDetParam.detTemplateId;
 
+  bool irradiationBiasCorrection_ = true;
```

Comment:
Workflows comparisons show some differences where I expected none. I think this line is the problem. Will test and update.